src/devtools/client/webconsole/actions/network.ts

```diff
@@ -11,6 +10,8 @@ import {
 let onResponseBodyPart: (responseBodyParts: responseBodyData) => void;
 
 export const setupNetwork = async (store: UIStore) => {
+  const ThreadFront = store.dispatch((dispatch, getState, { ThreadFront }) => ThreadFront);
```

Comment:
i'm not sure if i like this suggestion, but might be cleaner to shove this somewhere

kinda regret saying this, but maybe `const ThreadFront = store.getThreadFront()` this is gross, but doing the same thing and atleast `ThreadFront` isn't actually a global.

Alternatively, we could pass ThreadFront into `setupNetwork` which is fine in these cases

Reply:
Yeah, passing `ThreadFront` into these setup functions would also be an option, actually.  Runs the same either way, but at least it's more explicit.